app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.html

```diff
@@ -76,6 +89,29 @@
           </p>
         </div>
       </div>
+      <div class="row" ng-if="command.rollbackType === 'PREVIOUS_IMAGE'">
+        <div class="col-sm-11 col-sm-offset-1">
+          <ol>
+            <li>Deploy <em>{{ previousServerGroup.imageId }}</em> [
+              <strong>min</strong>: {{serverGroup.capacity.desired}},
+              <strong>max</strong>: {{ serverGroup.capacity.max }},
+              <strong>desired</strong>: {{ serverGroup.capacity.desired }}
+              ]<br/>(minimum capacity pinned at {{serverGroup.capacity.desired}} to prevent autoscaling down during rollback)
+            <li ng-if="command.rollbackContext.targetHealthyRollbackPercentage < 100">
+              Wait for at least {{minHealthy(command.rollbackContext.targetHealthyRollbackPercentage)}}
+              instances to report as healthy
+            </li>
+            </li>
+            <li>Disable {{ serverGroup.name }}</li>
+            <li>Restore minimum capacity of <em>new server group</em> [
```

Comment:
Is `Set` more appropriate than `Restore` here, since we're creating a new server group and the minimum capacity was never set the the `min` before?

Reply:
We're initially pinning the min capacity to something other than what it should be ... `Restore` still feels like reasonable wording if you ask me. 

FWIW it jives with the task name in orca as well `restoreMinCapacityFromSnapshot` which shows up when you do a normal red/black.

Reply:
Sounds good.